client/src/app/site/motions/modules/motion-detail/components/motion-comment/motion-comment.component.html

```diff
@@ -12,13 +12,11 @@
         </ng-container>
         <form [formGroup]="commentForm" *ngIf="isCommentEdited">
             <!-- The HTML Editor -->
-            <editor
+            <os-editor
                 [id]="section.id"
-                formControlName="comment"
+                _formControlName="comment"
                 [init]="tinyMceSettings"
-                (onInit)="onInitTinyMce($event)"
-                (onRemove)="onLeaveTinyMce($event)"
```

Comment:
These seem to have been a fragment from a previous restructure and even provoke errors since the functions do not exist anymore, so I removed them.